bin/qds.py

```diff
@@ -113,10 +114,10 @@ def submitaction(cmdclass, args):
         return 0
 
 
-def _getresult(cmdclass, cmd):
+def _getresult(cmdclass, cmd, args=[]):
     if Command.is_success(cmd.status):
         log.info("Fetching results for %s, Id: %s" % (cmdclass.__name__, cmd.id))
-        cmd.get_results(sys.stdout, delim='\t')
+        cmd.get_results(sys.stdout, delim='\t', qlog=cmd.qlog, arguments=args)
```

Comment:
cmd is the object which is returned when we run the command. So qlog is one of the attribute of that object same like cmd.status in above condition.